core/embed/trezorhal/stm32u5/secret.c

```diff
@@ -103,17 +135,7 @@ void secret_bhk_regenerate(void) {
 }
```

Comment:
just for total hygiene, `memzero(val)` at end too?

Reply:
https://github.com/trezor/trezor-firmware/commit/5bbddd37420c278a03dd8f333913b36b26eeb49d